dotnet-desktop-guide/framework/winforms/controls/how-to-create-an-unbound-windows-forms-datagridview-control.md

```diff
@@ -1,5 +1,8 @@
 ---
-title: Create an Unbound DataGridView Control
+title: How to Create an Unbound Windows Forms DataGridView Control
+
+description: Populate an unbound Windows Forms DataGridView Control programmatically and display a small amount of data in a table format without binding it to a data source
```

Comment:
@kirupap Right after I merged I noticed this blank line. Generally there aren't any blank lines between metadata items. Can you fix these up?